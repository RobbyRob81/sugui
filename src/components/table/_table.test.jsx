import * as React from 'react';
import { customRender } from '../../../test/test-utils';
import Table from '.';

const columns = [
  { id: 'id', header: 'ID' },
  { id: 'make', header: 'Make' },
  { id: 'model', header: 'Model' },
  { id: 'year', header: 'Year' },
  { id: 'type', header: 'Type' },
  { id: 'price', header: 'Price' }
];

it('should render without throwing an error', () => {
  const { container } = customRender(<Table columns={columns} />);
  expect(container.firstChild).toMatchSnapshot();
});
