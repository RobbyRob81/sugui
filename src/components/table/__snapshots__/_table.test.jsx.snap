// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render without throwing an error 1`] = `
<div
  class="sc-iwsKbI iuVdIU"
>
  <div
    class="sc-gZMcBi blBstE"
  >
    <div
      class="sc-cSHVUG gxPnBs sc-bZQynM dYKjUg"
    >
      <input
        class="sc-gzVnrw iYLwJF"
        color="porcelain"
        placeholder="Search the entire table..."
      />
      <div
        class="sc-dnqmqq ADIJr"
        color="porcelain"
      >
        <svg
          height="20"
          viewBox="0 0 20 20"
          width="20"
          xmlns="http://www.w3.org/2000/svg"
        >
          <g
            fill="none"
            fill-rule="evenodd"
          >
            <path
              d="M0 0h20v20H0z"
              fill="none"
              stroke="none"
            />
            <path
              d="M12.041 3.792a5.833 5.833 0 1 1-8.25 8.25 5.833 5.833 0 0 1 8.25-8.25M12.04 12.04l5.877 5.877"
              stroke-linecap="round"
              stroke-linejoin="round"
            />
          </g>
        </svg>
      </div>
    </div>
  </div>
  <div
    class="sc-gqjmRU dacZoj"
  >
    <table
      class="sc-VigVT bAAQEB"
    >
      <tbody />
    </table>
  </div>
</div>
`;
